# Pytis translations
# Copyright (C) 2006-2011 Brailcom, o.p.s.
# This file is distributed under the same license as the Pytis package.
# Paweł Loba <ploba@rogers.com>, 2006
#
msgid ""
msgstr ""
"Project-Id-Version: 0.1\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-01-13 17:02+0100\n"
"PO-Revision-Date: 2011-12-29 14:58+0100\n"
"Last-Translator: Pawel Loba <ploba@rogers.com>\n"
"Language-Team: Polish <wiking@lists.freebsoft.org>\n"
"Language: pl\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 "
"|| n%100>=20) ? 1 : 2;\n"

#: ../lib/pytis/data/access.py:434
msgid "Access denied"
msgstr ""

#: ../lib/pytis/data/dbapi.py:83
msgid "Can't connect to database"
msgstr ""

#: ../lib/pytis/data/dbapi.py:171
msgid "Database interface error"
msgstr ""

#: ../lib/pytis/data/dbapi.py:187
msgid "Database connection timeout"
msgstr ""

#: ../lib/pytis/data/dbapi.py:189 ../lib/pytis/data/dbapi.py:299
msgid "Database connection error"
msgstr ""

#: ../lib/pytis/data/dbapi.py:203 ../lib/pytis/data/pgsql.py:87
#: ../lib/pytis/data/pgsql.py:94
msgid "Database operational error"
msgstr ""

#: ../lib/pytis/data/dbapi.py:207 ../lib/pytis/data/pgsql.py:99
msgid "Database integrity violation"
msgstr ""

#: ../lib/pytis/data/dbdata.py:299
msgid "Too many database connections"
msgstr ""

#: ../lib/pytis/data/dbdata.py:695
msgid "Database error"
msgstr ""

#: ../lib/pytis/data/dbdata.py:763
#, fuzzy
msgid "Invalid user name or password"
msgstr "Niewłaściwy format daty lub czasu"

#: ../lib/pytis/data/pgsql.py:60
msgid "Unable to connect to the database"
msgstr ""

#: ../lib/pytis/data/pgsql.py:69
msgid "Query arguments unsupported by the pgsql backend"
msgstr ""

#: ../lib/pytis/data/postgresql.py:275 ../lib/pytis/data/postgresql.py:361
#: ../lib/pytis/data/postgresql.py:373
msgid "Method not implemented"
msgstr ""

#: ../lib/pytis/data/postgresql.py:508
msgid "Characters present in data not representable in database encoding"
msgstr ""

#: ../lib/pytis/data/postgresql.py:3669
#, fuzzy
msgid "Invalid database counter value"
msgstr "Niewłaściwy format daty lub czasu"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:162
msgid "Empty value"
msgstr "Pusta wartość"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:164
#, fuzzy
msgid "Invalid value"
msgstr "Niepoprawny numer"

#: ../lib/pytis/data/types_.py:396
#, python-format
msgid "Wildcard matching not supported for values of type '%s'."
msgstr ""

#. User input validation error message.
#: ../lib/pytis/data/types_.py:530
#, python-format
msgid "Minimal value is %(minimum)s"
msgstr ""

#. User input validation error message.
#: ../lib/pytis/data/types_.py:533
#, python-format
msgid "Maximal value is %(maximum)s"
msgstr ""

#. User input validation error message.
#: ../lib/pytis/data/types_.py:610
#, fuzzy, python-format
msgid "Minimal size %(minlen)s not satisfied"
msgstr "Ciąg przekracza maksymalną długość %(maxlen)d"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:613
#, fuzzy, python-format
msgid "Maximal length %(maxlen)s characters exceeded"
msgstr "Ciąg przekracza maksymalną długość %(maxlen)d"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:715
msgid "Not an integer"
msgstr "To nie jest liczba całkowita"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:825
msgid "Invalid number"
msgstr "Niepoprawny numer"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:959
#, fuzzy, python-format
msgid "Minimal length %(minlen)s characters not satisfied"
msgstr "Ciąg przekracza maksymalną długość %(maxlen)d"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:961 ../lib/pytis/data/types_.py:2010
#, fuzzy, python-format
msgid "Maximal size %(maxlen)s exceeded"
msgstr "Ciąg przekracza maksymalną długość %(maxlen)d"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1051
msgid "Enter the password twice to eliminate typos"
msgstr ""

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1054
msgid "Passwords don't match"
msgstr ""

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1057
#, fuzzy
msgid "Invalid MD5 hash"
msgstr "Niepoprawna wartość"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1060
msgid "Please use mix of letters and non-letters in your password"
msgstr ""

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1122 ../lib/pytis/data/types_.py:1156
#: ../lib/pytis/data/types_.py:1192 ../lib/pytis/data/types_.py:1821
msgid "Invalid format"
msgstr "Niewłaściwy format"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1142
msgid "Invalid color format ('#RGB' or '#RRGGBB')"
msgstr "Niewłaściwy format koloru ('#RGB' lub '#RRGGBB')"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1158
#, python-format
msgid "Invalid inet address mask: %(mask)s"
msgstr ""

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1160
#, python-format
msgid "Invalid inet address value %(addr)s"
msgstr ""

#: ../lib/pytis/data/types_.py:1318
msgid "Invalid date or time format"
msgstr "Niewłaściwy format daty lub czasu"

#: ../lib/pytis/data/types_.py:1319
msgid "Invalid date or time"
msgstr "Niewłaściwy format daty lub czasu"

#: ../lib/pytis/data/types_.py:1320
msgid "Date outside the allowed range"
msgstr "Data poza dopuszczalnym obrębem"

#: ../lib/pytis/data/types_.py:1975
msgid "Invalid boolean type input value"
msgstr ""

#: ../lib/pytis/data/types_.py:2056
#, python-format
msgid "Not a buffer object: %r"
msgstr ""

#: ../lib/pytis/data/types_.py:2192
#, fuzzy, python-format
msgid "Maximal pixel size %(maxsize)s exceeded"
msgstr "Ciąg przekracza maksymalną długość %(maxlen)d"

#: ../lib/pytis/data/types_.py:2194
#, fuzzy, python-format
msgid "Minimal pixel size %(minsize)s exceeded"
msgstr "Ciąg przekracza maksymalną długość %(maxlen)d"

#: ../lib/pytis/data/types_.py:2196
#, python-format
msgid "Unsupported format %(format)s; valid formats: %(formats)s"
msgstr ""

#: ../lib/pytis/data/types_.py:2212
#, fuzzy
msgid "Invalid graphics format"
msgstr "Niewłaściwy format"

#: ../lib/pytis/data/types_.py:2317
#, fuzzy
msgid "Invalid hierarchical value format"
msgstr "Niewłaściwy format daty lub czasu"

#: ../lib/pytis/data/types_.py:2319
msgid "One of hierarchical value items is too long"
msgstr ""

#: ../lib/pytis/data/types_.py:2321
msgid "One of hierarchical value items contains invalid characters"
msgstr ""

#, fuzzy
#~ msgid "String exceeds max length %(maxlen)s characters"
#~ msgstr "Ciąg przekracza maksymalną długość %(maxlen)d"

#~ msgid "Submit"
#~ msgstr "Wyślij"

#~ msgid "Yes"
#~ msgstr "Tak"

#~ msgid "No"
#~ msgstr "Nie"

#~ msgid "Reset"
#~ msgstr "Odśwież"

#, fuzzy
#~ msgid "Sum"
#~ msgstr "Wyślij"

#~ msgid "No records."
#~ msgstr "Nie ma wpisu."

#, fuzzy
#~ msgid "Total records:"
#~ msgstr "Nie ma wpisu."

#, fuzzy
#~ msgid "Instructions"
#~ msgstr "Działania"

#, fuzzy
#~ msgid "Selections"
#~ msgstr "Działania"

#, fuzzy
#~ msgid "Dictation"
#~ msgstr "Działania"

#~ msgid "reduced"
#~ msgstr "zredukowany"
